lib/cli/src/initiate.ts

```diff
@@ -326,8 +327,10 @@ export function initiate(options: CommandOptions, pkg: Package): Promise<void> {
   }
   done();
 
-  return installStorybook(projectType, {
+  await installStorybook(projectType, {
     ...options,
     ...(isEsm ? { commonJs: true } : undefined),
   });
+
+  return automigrate({ yes: true });
```

Comment:
I'm skeptical of `yes: true`. Perhaps we should prompt the user instead so that if something goes wrong they know where to look?

Reply:
Makes sense. Fixed!